src/pages/guides/getting_started/hello-world.md

```diff
@@ -46,28 +46,40 @@ The [Command Line Interface (CLI)](#command-line-interface-cli) path will teach
 - Basic familiarity with JavaScript, HTML and CSS.
 - Node.js version 18 or higher (optional, only for the CLI track).
 
-### Add-on Development mode
+### Enable Add-on Development Mode
 
-A pre-requisite for both tracks is to have enabled the **Add-on Development** mode. Open Adobe Express in the browser and see the following animation for instructions, or expand the details below for a step-by-step guide.
+Before you can build add-ons, you need to enable Add-on Development mode in Adobe Express—you only need to do this once.
 
-![Enable Add-on Development](./img/playground-enable-dev-mode.gif)
+<InlineAlert slots="text" variant="success"/>
+
+This is **automatically done** when you launch the Code Playground [from this link](https://www.adobe.com/go/addon-playground), when you connect to your local add-on development environment via [this link](https://www.adobe.com/go/addon-cli), or when you click the buttons you'll find later in this guide.
```

Comment:
'Automatically done' makes it sound as if the dev mode will be enabled by automatically when devs access the playground or addon-testing via the URL. Can we rephrase this to 'The first time you launch the [Code Playground](https://www.adobe.com/go/addon-playground) or [connect to your local add-on development environment](https://www.adobe.com/go/addon-cli), you can review the Developer Terms of Use and enable Developer mode.